content/html/concepts/comments/comments.md

```diff
@@ -0,0 +1,41 @@
+---
+Title: 'Comments'
+Description: 'HTML comments are markup that are included in the HTML source, but do not affect the rendering of the HTML document.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Comments'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+HTML comments are markup that are included in the HTML source, but do not affect the rendering of the HTML document. They are used for documenting the HTML code and for "commenting out" HTML that the developer doesn't want to render, but does not desire to delete, generally for historical or debugging purposes.
+
+## Syntax
+
+```html
+<!-- comment goes here -->
+```
+
+An HTML comment can contain any other HTML markup between the opening `<!--` and the closing `-->` except for another comment. Attempting to nest HTML comments will result in unexpected behavior because the comment is closed at the first `-->` it encounters. All text after the first `-->` will render, including the outer enclosing `-->`. Thus, when commenting out HTML code, the developer must make sure any interior comments are removed or strip the trailing `-->`s.
```

Comment:
```suggestion
An HTML comment can contain any other HTML markup between the opening `<!--` and the closing tag `-->` except for another comment. Attempting to nest HTML comments will result in unexpected behavior because the comment is closed at the first `-->` it encounters. All text after the first `-->` will render, including the outer enclosing `-->`. Thus, when commenting out HTML code, the developer must make sure any interior comments are removed or strip the trailing `-->`s.
```